applications/nublado/templates/controller-ingress-files.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
```

Comment:
If this doesn't need to be usable as a separate WebDAV route (with basic auth and whatnot), you can merge this with `controller-ingress-user.yaml` as an additional route and save the overhead of having another ingress.